### PR TITLE
Change markdown to accommodate config_flow setup

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Change markdown for `trafikverket_weatherstation` for config_flow

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/60078 https://github.com/home-assistant/core/pull/60664
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/trafikverket_weatherstation.markdown [hunk 1]
```diff
@@ -14,75 +15,25 @@ ha_platforms:
...
-      description: Average wind speed during the last 10 minutes.
-    wind_speed_max:
-      description: Maximum wind speed measured during the last 30 minutes.
-    precipitation_amount:
-      description: Amount of precipitation.
-    precipitation_amountname:
-      description: Amount of precipitation in thirteen variants (Swedish text).
-    
-{% endconfiguration %}
+## Monitored conditions can be one of many of
```
Comment (frenck) [inline source/_integrations/trafikverket_weatherstation.markdown]: Monitored conditions are no longer allowed, see:

<https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/trafikverket_weatherstation.markdown (hunk 1 above)
Comment (gjohansson-ST) [reply source/_integrations/trafikverket_weatherstation.markdown]: Hi. In the config flow the user is presented with all options in a user friendly manner so he can choose and doesn't need to know the options available. Should I just call it something different instead to make this clear?

Comment (frenck) [reply source/_integrations/trafikverket_weatherstation.markdown]: > In the config flow the user is presented with all options in a user friendly manner so he can choose and doesn't need to know the options available

That is no longer allowed.

> Should I just call it something different instead to make this clear?

No, just expose all available entities instead. Home Assistant provides options for users to enable/disable entities nowadays.

Comment (gjohansson-ST) [reply source/_integrations/trafikverket_weatherstation.markdown]: Clear! I will take care

Comment (gjohansson-ST) [reply source/_integrations/trafikverket_weatherstation.markdown]: Very clear and now done so also then PR https://github.com/home-assistant/core/pull/60664 needs to be approved. Will update this markdown accordingly

File: source/_integrations/trafikverket_weatherstation.markdown [hunk 2]
```diff
@@ -14,75 +15,25 @@ ha_platforms:
 
 Showing weather information provided by [Trafikverket](https://www.trafikverket.se/) weather stations in Sweden.
 
-Potential use cases:
+## Potential use cases:
```
Comment (frenck) [inline source/_integrations/trafikverket_weatherstation.markdown]:
```suggestion
## Potential use cases
```

File: source/_integrations/trafikverket_weatherstation.markdown [hunk 3]
```diff
@@ -14,75 +15,25 @@ ha_platforms:
...
+- Relative humidity.
+- Type of precipitation (Swedish text).
+- Wind direction in degrees.
+- Rough wind direction in twelve variants (Swedish text).
+- Average wind speed during the last 10 minutes.
+- Maximum wind speed measured during the last 30 minutes.
+- Amount of precipitation.
+- Amount of precipitation in thirteen variants (Swedish text).
 
 ## Obtaining API key
```
Comment (frenck) [inline source/_integrations/trafikverket_weatherstation.markdown]: Let move this above the config flow setup instructions (as one would need it before doing that); and lets name the chapter "Prerequisites" to match other integrations.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @gjohansson-ST 👍

**Pull request merged**